lib/db.ts

```diff
@@ -481,6 +481,53 @@ export async function updateUserConfig(
   return newConfig
 }
 
+export async function getUsageByUserStats(
+  conn: ServerlessMysql,
+  githubUser: Session['user']
+): Promise<any> {
+  const isAdmin = await isAdminUser(conn, githubUser.login)
+  if (!isAdmin) {
+    throw new Error('Not available')
+  }
+
+  // FIXME change to admin != 1
```

Comment:
**Nit:** Seems like this should actually be fixed before merging?

Reply:
I'll put in a flag so that in prod, it ignores admins, but everywhere else, admins are included.  Reason is that in dev and staging we're all (or most of us are) admins.